src/rpcblockchain.cpp

```diff
@@ -706,9 +709,10 @@ UniValue getblockchaininfo(const UniValue& params, bool fHelp)
     bip9_softforks.push_back(BIP9SoftForkDesc("segwit", consensusParams, Consensus::DEPLOYMENT_SEGWIT));  // MVF-BU added
     obj.push_back(Pair("softforks",             softforks));
     obj.push_back(Pair("bip9_softforks", bip9_softforks));
+    obj.push_back(Pair("difficultyadjinterval", consensusParams.DifficultyAdjustmentInterval(tip->nHeight))); // MVF-BU
 
     // MVF-BU begin output hardfork description (MVHF-BU-DES-TRIG-9)
-    if (!isMVFHardForkActive)
+    if (!isMVFHardForkActive && consensusParams.MVFisWithinRetargetPeriod(tip->nHeight))
```

Comment:
I added line 712 to push difficultyadjinterval but don't know why this line was changed. It could be updated as follows:
!isMVFHardForkActive || consensusParams.MVFisWithinRetargetPeriod(tip->nHeight) 
isMVFHardForkActive && consensusParams.MVFisWithinRetargetPeriod(tip->nHeight)
!isMVFHardForkActive

Seems to me it should be the last one?